core/templates/dev/head/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.html

```diff
@@ -140,15 +140,23 @@
           </a>
           <div ng-if="activeHintIndex === $index">
             <div class="oppia-editor-card-section protractor-test-hint-<[$index]>-text">
-              <angular-html-bind html-data="getRequiredHtml(hint.hintContent)" class="oppia-rte-editor">
-              </angular-html-bind>
+              <span ng-show="!getRequiredHtml(hint.hintContent)" class="oppia-placeholder">
+                <[getEmptyContentMessage()]>
+              </span>
+              <span>
```

Comment:
I know the `html-bind` below wouldn't display anything if it is empty, but can you put this `span` also in an `ng-if=getRequiredHtml(..)`, so that we can be sure that the directive below is not called in this case?

Reply:
Ok I'll make sure the other tabs also have not been changed by this commit to use 'ng-if'.

Reply:
I don't think this is done. I meant, add `ng-if=getRequiredHtml(..)` to this `span`, so that we can be sure this span is not rendered at all when it's not needed.

Reply:
So '<span ng-if="!getRequiredHtml(hint.hintContent)" class="oppia-placeholder">' ? Or is it changing the span that is one after this one?

Reply:
Ok I changed it to be ng-if so done.